### PR TITLE
docs(apisix): proofread GET APISIX, Configure Routes, Load Balancing; plus a few formatting updates 

### PR DESCRIPTION
**Pull request by Kayx23**

### CODE REVIEW

File: apisix_versioned_docs/version-3.1.0/getting-started/README.md [hunk 1]
```diff
@@ -24,15 +24,15 @@ To provide a better experience in this tutorial, the authorization of Admin API
 
 :::
         
-The APISIX can be easily installed and started with the quickstart script.
+APISIX can be easily installed and started with the quickstart script:
 
-```sh
+```shell
```
Comment (juzhiyuan) [inline apisix_versioned_docs/version-3.1.0/getting-started/README.md:29]: bash?

File: apisix_versioned_docs/version-3.1.0/getting-started/README.md [hunk 2]
```diff
@@ -24,15 +24,15 @@ To provide a better experience in this tutorial, the authorization of Admin API
...
 curl -sL https://run.api7.ai/apisix/quickstart | sh
 ```
 
-This command runs APISIX and etcd locally with Docker. APISIX uses etcd to save and synchronize configuration. Both the etcd and the APISIX use [**host**](https://docs.docker.com/network/host/) Docker network mode. That is, the APISIX can be accessed from local.
+The script should start two docker containers, _apisix-quickstart_ and _etcd_. APISIX uses etcd to save and synchronize configurations. Both the etcd and the APISIX use [**host**](https://docs.docker.com/network/host/) docker network mode. That is, the APISIX can be accessed from local.
```
Comment (juzhiyuan) [inline apisix_versioned_docs/version-3.1.0/getting-started/README.md:33]: Where is the `script`?

Comment (juzhiyuan) [inline apisix_versioned_docs/version-3.1.0/getting-started/README.md:33]:
```suggestion
The script should start two Docker containers, _apisix-quickstart_ and _etcd_. APISIX uses etcd to save and synchronize configurations. Both the etcd and the APISIX use [**host**](https://docs.docker.com/network/host/) docker network mode. That is, the APISIX can be accessed from local.
```

File: apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md [hunk 3]
```diff
@@ -3,36 +3,36 @@ title: Configure Routes
 slug: /getting-started/configure-routes
 ---
 
-Apache APISIX provides flexible gateway management capabilities based on _Route_, which defines the request's routing path and targets.
+Apache APISIX provides flexible gateway management capabilities based on _Routes_, where routing paths and targets are defined for requests.
```
Comment (juzhiyuan) [inline apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md:6]: active voice vs passive voice?

File: apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md [hunk 4]
```diff
@@ -3,36 +3,36 @@ title: Configure Routes
...
 
-Apache APISIX provides flexible gateway management capabilities based on _Route_, which defines the request's routing path and targets.
+Apache APISIX provides flexible gateway management capabilities based on _Routes_, where routing paths and targets are defined for requests. 
 
-This tutorial guides you on how to create one Route and validate it. You will complete the following steps:
+This tutorial guides you on how to create a route and validate it. You will complete the following steps:
 
-1. Create a Route with an example _Upstream_, which points to [httpbin.org](http://httpbin.org).
-2. Use _cURL_ to send a test request, showing how APISIX proxies and forwards the request.
+1. Create a route with a sample _upstream_ that points to [httpbin.org](http://httpbin.org).
```
Comment (juzhiyuan) [inline apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md:10]: The initial goal to use `Route` and `Upstream` is to express `APISIX Resource Object/Entity`, should we use `Lowercase`?

File: apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md [hunk 5]
```diff
@@ -3,36 +3,36 @@ title: Configure Routes
...
+* Complete [Get APISIX](./) to install APISIX.
 
-## What's a Route
+## What is a Route
 
-A Route is a routing path to the upstream targets. In [Apache APISIX](https://api7.ai/apisix), the Route matches the client's requests based on defined rules, loads and executes the corresponding plugins, then forwards the request to the specified upstream targets.
+A route is a routing path to upstream targets. In [Apache APISIX](https://api7.ai/apisix), routes are responsible for matching client's requests based on defined rules, loading and executing the corresponding plugins, as well as forwarding requests to the specified upstream services.
 
-One Route should have _uri_ and the corresponding Upstream at least.
+A route should have an _uri_ and a corresponding upstream at the minimum.
```
Comment (juzhiyuan) [inline apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md:21]: This sentence mainly expresses: `uri` and `upstream` parameters are required. Should we use `minimum`?

File: apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md [hunk 6]
```diff
@@ -3,36 +3,36 @@ title: Configure Routes
...
-An Upstream is a set of target nodes with the same work. The Upstream defines the virtual host abstraction that performs load balancing on a given set of service nodes according to the configured rules.
+An upstream is a set of target nodes with the same work. It defines a virtual host abstraction that performs load balancing on a given set of service nodes according to the configured rules.
 
 ## Create a Route
 
-In this section, let's create a Route to forward client requests to the [httpbin.org](http://httpbin.org), which is a public HTTP Request and Response service.
+In this section, you will create a route that forwards client requests to [httpbin.org](http://httpbin.org), a public HTTP request and response service.
 
-The following command creates one Route which matches all requests to the `http://127.0.0.1:9080/ip` endpoint, forwards them to the [httpbin.org/ip](http://httpbin.org/ip), and returns the requester's IP address.
+The following command creates a route, which should forward all requests sent to `http://127.0.0.1:9080/ip` to [httpbin.org/ip](http://httpbin.org/ip):
```
Comment (juzhiyuan) [inline apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md:31]: `to xxx to xxx`?

File: apisix_versioned_docs/version-3.1.0/getting-started/load-balancing.md [hunk 7]
```diff
@@ -3,22 +3,22 @@ title: Load Balancing
 slug: /getting-started/load-balancing
 ---
 
-Load balancing manages traffic between clients and servers, it decides which server will handle a specific request, and it allows for improved performance, scalability, and reliability. It's essential for any servers that need to process large volume of traffic. 
+Load balancing manages traffic between clients and servers. It is a mechanism used to decide which server handles a specific request, allowing for improved performance, scalability, and reliability. Load balancing a key consideration in designing systems that need to handle a large volume of traffic.
```
Comment (juzhiyuan) [inline apisix_versioned_docs/version-3.1.0/getting-started/load-balancing.md:6]:
```suggestion
Load balancing manages traffic between clients and servers. It is a mechanism used to decide which server handles a specific request, allowing for improved performance, scalability, and reliability. Load balancing is a key consideration in designing systems that need to handle a large volume of traffic. 
```

File: apisix_versioned_docs/version-3.1.0/getting-started/load-balancing.md [hunk 8]
```diff
@@ -3,22 +3,22 @@ title: Load Balancing
...
 
-In this tutorial, you will create one Route with two upstream services, then enable round-robin load balancing to distribute traffic between the two services.
+In this tutorial, you will create a route with two upstream services and enable round-robin load balancing to distribute traffic between the two services.
 
-## Prerequisites
+## Prerequisite(s)
 
-1. Complete the [Get APISIX](./) to install APISIX first.
-2. Understand APISIX [Route and Upstream](./configure-routes#whats-a-route).
+* Complete [Get APISIX](./) to install APISIX.
```
Comment (juzhiyuan) [inline apisix_versioned_docs/version-3.1.0/getting-started/load-balancing.md:14]: Unordered List vs Ordered List?

File: apisix_versioned_docs/version-3.1.0/getting-started/load-balancing.md [hunk 9]
```diff
@@ -76,18 +74,18 @@ From `mock.api7.ai`:
...
-Let's generate 100 requests to test the load-balancing effect.
+Let's generate 100 requests to test the load-balancing effect:
 
-```sh
+```shell
 hc=$(seq 100 | xargs -i curl "http://127.0.0.1:9080/headers" -sL | grep "httpbin" | wc -l); echo httpbin.org: $hc, mock.api7.ai: $((100 - $hc))
 ```
 
-The results are as follows, the response changes between the two services, and the number of responses from the two services are close.
+The result shows the requests were distributed over the two services almost equally:
```
Comment (juzhiyuan) [inline apisix_versioned_docs/version-3.1.0/getting-started/load-balancing.md:83]: Can pick this change: https://github.com/api7/docs/pull/38/files#diff-254cb062850b22744b44e3666741e0c9183126cc4b3121cf176f4c79db055fbeR88

File: apisix_versioned_docs/version-3.1.0/getting-started/README.md (hunk 1 above)
Comment (Kayx23) [reply apisix_versioned_docs/version-3.1.0/getting-started/README.md:29]: I changed it to shell to be consistent with API7 Cloud's doc src. They use `shell` and for some reason it is rendering.

File: apisix_versioned_docs/version-3.1.0/getting-started/README.md (hunk 2 above)
Comment (Kayx23) [reply apisix_versioned_docs/version-3.1.0/getting-started/README.md:33]: It refers to the quickstart script.

File: apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md (hunk 3 above)
Comment (Kayx23) [reply apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md:6]: discussion

File: apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md (hunk 4 above)
Comment (Kayx23) [reply apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md:10]: They are also entities/objects in Kong and not capitalized: 
* `Upstream`: https://docs.konghq.com/gateway/latest/key-concepts/upstreams/
* `Service` and `Route`: https://docs.konghq.com/gateway/latest/get-started/services-and-routes/

File: apisix_versioned_docs/version-3.1.0/getting-started/README.md (hunk 2 above)
Comment (Kayx23) [reply apisix_versioned_docs/version-3.1.0/getting-started/README.md]: committed

File: apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md (hunk 5 above)
Comment (Kayx23) [reply apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md:21]: if we're going to say `at least`, `at the minimum` sounds a lot better imho. Alternative we can just say "A route should have an _uri_ and a corresponding upstream"?

File: apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md (hunk 6 above)
Comment (Kayx23) [reply apisix_versioned_docs/version-3.1.0/getting-started/configure-routes.md:31]: sounds ok to me

File: apisix_versioned_docs/version-3.1.0/getting-started/load-balancing.md (hunk 8 above)
Comment (Kayx23) [reply apisix_versioned_docs/version-3.1.0/getting-started/load-balancing.md:14]: unordered list was used in GET APISIX (or README.md in the source) so it was inconsistent across the existing docs. I changed them all to unordered.

File: apisix_versioned_docs/version-3.1.0/getting-started/load-balancing.md (hunk 9 above)
Comment (Kayx23) [reply apisix_versioned_docs/version-3.1.0/getting-started/load-balancing.md:83]: I followed this tutorial and the result wasn't 50/50. To be discussed.

File: apisix_versioned_docs/version-3.1.0/getting-started/load-balancing.md (hunk 8 above)
Comment (Kayx23) [reply apisix_versioned_docs/version-3.1.0/getting-started/load-balancing.md]: fixed

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Comment by An-DJ** [comment]
LGTM

**Review by An-DJ — APPROVED** [review]

**Pull request merged**